MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,15 +20,30 @@ final class FeedViewModel {
 	
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onErrorStateChange: Observer<String?>?
+	
 	
 	func loadFeed() {
+		self.onErrorStateChange?(nil)
```

Comment:
`self.` can be omitted here:

```suggestion
		onErrorStateChange?(nil)
```